config/initializers/sentry.rb

```diff
@@ -1,4 +1,6 @@
-Raven.configure do |config|
-  config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
-  config.transport_failure_callback = ->(event) { Rails.logger.error(event.to_s) }
-end
+if Rails.env.production? 
+  Raven.configure do |config|
+    config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
+    config.transport_failure_callback = ->(event) { Rails.logger.error(event.to_s) }
+  end
+end
```

Comment:
```suggestion
Raven.configure do |config|
  config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
  config.transport_failure_callback = ->(event) { Rails.logger.error(event.to_s) }
  config.environments = %w(production)
end	    
```

So from what I remember the errors are just that Raven hasn't got a DSN set so can't send the errors. If we don't want this noisy output in development I _think_ it's possible to just set `config.environments = %w(production)` as above.

Reply:
Thank you for this... this solved the issues of log messages not coming through.